### PR TITLE
Add SCMP_ACT_LOG as a valid Seccomp action

### PR DESCRIPTION
**Pull request by blacktop**
This enables logging of all system calls in a container. This could be useful for creating minimal seccomp profiles, and is the only Seccomp action runc doesn't support at present.

### CODE REVIEW

**Comment by blacktop** [comment]
REF https://github.com/moby/moby/issues/38333

**Comment by blacktop** [comment]
This is failing because it requires https://github.com/seccomp/libseccomp-golang/pull/29 to be merged 😢

**Comment by blacktop** [comment]
# 😁 🎉

**Comment by thaJeztah** [comment]
@justincormack PTAL

**Comment by blacktop** [comment]
@justincormack do you need me to do anything else?

**Comment by blacktop** [comment]
ping @justincormack

**Comment by blacktop** [comment]
ping @justincormack

File: Dockerfile [hunk 1]
```diff
@@ -67,4 +67,4 @@ ENTRYPOINT ["/tmpmount"]
 ADD . /go/src/github.com/opencontainers/runc
 
 RUN . tests/integration/multi-arch.bash \
-    && curl -o- -sSL `get_busybox` | tar xfJC - ${ROOTFS}
+    && curl -o- -sSL `get_busybox` | tar xfJC - ${ROOTFS}
```
Comment (thaJeztah) [inline Dockerfile]: Looks like this inadvertently removed a newline

Comment (blacktop) [reply Dockerfile]: I have removed them and learned more about git squashing ;)

**Comment by thaJeztah** [comment]
Thanks for squashing 👍 
I'm not a maintainer in this repository, but I noticed that you used your GitHub handle in your sign-off; 

```
Signed-off-by: blacktop <blacktop@users.noreply.github.com>
```

Perhaps you can change to your real name? (see [contributing.md](https://github.com/opencontainers/runc/blob/master/CONTRIBUTING.md#sign-your-work);

> using your real name (sorry, no pseudonyms or anonymous contributions.)


Changes themselves SGTM, but I'll try to get @justincormack and/or @crosbymichael to have a look

**Comment by crosbymichael** [comment]
LGTM

**Comment by pjbgf** [comment]
@crosbymichael @thaJeztah @justincormack is there anything else left before we can merge this?

**Comment by AkihiroSuda** [comment]
@cyphar @mrunalp PTAL

**Comment by blacktop** [comment]
@crosbymichael @thaJeztah @justincormack I have rebased so we should be good to go now?

**Comment by thaJeztah** [comment]
Looks like there's a merge-commit in your branch; did you do a rebase, or a merge?

**Comment by mrunalp** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Comment by blacktop** [comment]
We did it!!!! 
# 👍 😎 👍

**Pull request merged**